### DISCUSSION TITLE
Can kamal be used for running bunch of services locally in development?

### DISCUSSION THREAD

**Discussion by sumanp** [basecamp/kamal#1123] [Q&A]
We have a few micro-services that needs to be run locally to instantiate the entire application. We can do it with docker-compose but its a huge pain. Plus we don't want docker desktop dependency due to large memory footprint. Can I use Kamal for rapid local dev?

**Comment by sasharevzin** [comment]
it feels pointless, but you can always run Ubuntu locally and deploy to the container's IP  ¯\_(ツ)_/¯ 
```docker run -ti --rm ubuntu /bin/bash```

**Reply by sumanp** [reply]
It seems impractical but I get what you mean. We just to implement what docker-compose does on dev, with Kamal. Something similar to `docker-compose up`

Is that currently posible with Kamal?

**Comment by nanaya** [comment]
not answering the kamal question but if you're using linux, [docker compose plugin](https://docs.docker.com/compose/install/linux/#install-using-the-repository) is a thing and doesn't need the full desktop install.
